shared/src/vyxal/Elements.scala

```diff
@@ -1071,8 +1071,8 @@ object Elements:
     addPart(
       Dyad,
       "İ",
-      "Drop | Collect While Unique | Complex Number",
-      List("drop", "collect-while-unique", "complex"),
+      "Zero Slice From | Collect While Unique | Complex Number",
+      List("zero-slice-from", "slice-from", "from", "drop-from", "collect-while-unique", "complex"),
```

Comment:
```suggestion
      "Drop | Collect While Unique | Complex Number",
      List("zero-slice-from", "slice-from", "drop", "collect-while-unique", "complex"),
```
The `zero-slice-from` and `slice-from` keywords are a good idea, but `drop` is also a common name for this operation (also, I've never seen anyone call it `drop-from`)

Reply:
Personally, I've never heard of `drop`. I like the keywords, but maybe we could call it "Zero Slice From/Drop"?

Reply:
Drop is definitely the most common name, for example Scala, Haskell, and APL all call it "drop".

Reply:
@RubenVerg Alright, I've changed it to Drop/Zero Slice From.